packages/styles/components/_c.notification.scss

```diff
@@ -0,0 +1,109 @@
+$text-font-size: get-token(size, font, '04');
+$title-font-size: get-token(size, font, '05');
+
+.mc-notification {
+  box-sizing: border-box;
+  position: relative;
+  display: block;
+  border-width: get-border('s');
+  border-style: solid;
+  font-size: $text-font-size;
+  width: 288px;
+  padding: $mu150 $mu150 $mu150 $mu400;
+
+  @include set-border-radius();
+  @include notification-border-background();
+  @include set-font-face();
+
+  &,
+  & a,
+  & button,
+  & h3 {
+    @include set-color('darkest');
+  }
+
+  & h3,
+  & p,
+  & button {
+    margin: 0;
+  }
```

Comment:
I forgot to tell you that we only use styles on elements with a class. So you have to add many classes, sorry.